schwifty/registry.py

```diff
@@ -41,7 +44,10 @@ def has(name: str) -> bool:
 def get(name: str) -> Union[Dict, List[Dict]]:
     if not has(name):
         data = None
-        directory = files(__package__) / f"{name}_registry"
+        package_path = files(__package__)
```

Comment:
I guess to fix the typing issue that `mypy` complains about you have to
```python
try:
    from importlib.abc import Traversable
    from importlib.resources import files
except ImportError:
    from importlib_resources import files  # type: ignore
    from importlib_resources.abc import Traversable  # type: ignore
```
and
```python
def get(name: str) -> Union[Dict, List[Dict]]:
        ...
        package_path: Union[Traversable, Path]
        if getattr(sys, "frozen", False) and hasattr(sys, "_MEIPASS"):
            package_path = Path(getattr(sys, "_MEIPASS"))
        else:
            package_path = files(__package__)
```

Reply:
That works, thx!